### ISSUE TITLE
Seems outdated

### ISSUE THREAD

**Issue by thebigsmileXD** [svilex/PocketBurger#1]
Seems to be outdated. also i get real formatting bugs on windows (textFormat) 
![image](https://cloud.githubusercontent.com/assets/8733998/18087349/0b1b6d92-6eb5-11e6-8c27-1883cb9ba2c1.png)

[results.txt](https://github.com/svilex/PocketBurger/files/444701/results.txt)

**Comment by svilex** [comment]
Yes it is. Formatting bugs are because you need to enable ansi. I don't think i will update it soon.
